go/vt/sidecardb/schema/schemaengine/tables.sql

```diff
@@ -16,8 +16,8 @@ limitations under the License.
 
 CREATE TABLE IF NOT EXISTS tables
 (
-    TABLE_SCHEMA varchar(64) NOT NULL,
-    TABLE_NAME varchar(64) NOT NULL,
+    TABLE_SCHEMA varchar(64) CHARACTER SET `utf8mb4` COLLATE `utf8mb4_0900_bin` NOT NULL,
+    TABLE_NAME varchar(64) CHARACTER SET `utf8mb4` COLLATE `utf8mb4_0900_bin` NOT NULL,
```

Comment:
I deliberately suggested utf8mb3 since that still is used for internal system tables for MySQL, even though it’s deprecated. 

The problem of this collation is that it’s not available for MySQL 5.7.